exercises/affine-cipher/example.R

```diff
@@ -0,0 +1,63 @@
+# encrypts message
+encrypt <- function(message, a, b) {
+  m <- 26
+
+  # computes the greatest common divisor of numbers x and y
+  gcd <- function(x, y) {
+    r <- x %% y
+    return(ifelse(r, gcd(y, r), y))
+  }
+
+  # must check a and m are coprime
+  if (gcd(a, m) != 1) {
+    stop("a and 26 must be co-prime")
+  }
+
+  # removed whitespace & lower-cased
+  parsedmessage <- tolower(gsub(" ", "", message))
+  # list of letters
+  splitlist <- strsplit(parsedmessage, "")[[1]]
+  # index of letters
+  x <- match(splitlist, letters) - 1
+
+  # E(x) = (ax + b) mod m
+  return(paste(letters[ ((a * x + b) %% m) + 1], collapse = ""))
+}
+
+# decrypts encryption
+decrypt <- function(encryption, a, b) {
+  m <- 26
+
+  # computes the greatest common divisor of numbers x and y
+  gcd <- function(x, y) {
+    r <- x %% y
+    return(ifelse(r, gcd(y, r), y))
+  }
+
+  # must check a and m are coprime
+  if (gcd(a, m) != 1) {
+    stop("a and 26 must be co-prime")
+  }
```

Comment:
Duplicate of L11-14, see https://github.com/exercism/r/pull/125/files#r312707113